Makefile

```diff
@@ -1,13 +1,54 @@
-SHELL := /bin/bash
+SPACE := $(subst , ,)
+PACK = $(subst $(SPACE),,$(1))
+CMD_NOT_FOUND = $(error $(call PACK, $(1)) is required for this rule)
 
-GREEN := \033[32m
-RESET := \033[39m
-ARROW := \033[1m\033[31m>$(GREEN)>\033[33m>$(RESET)
+CHECK_CMD = $(if $(shell command -v $(1)),, $(call CMD_NOT_FOUND, $(1)))
```

Comment:
Function that check for a command to exist.
If the check fails, it will print the error message `CMD_NOT_FOUND` and quit.

Example: `$(call CMD_NOT_FOUND, npm)`